docs/patterns/triggers/triggers.mdx

```diff
@@ -39,6 +45,24 @@ toolset = ComposioToolSet()
 trigger_schema = toolset.get_trigger("GITHUB_STAR_ADDED_EVENT")
 print(trigger_schema.config)
 ```
+
+Response:
+```json
+properties={
```

Comment:
The JSON response format could be improved for better readability:
1. Use double quotes instead of single quotes for JSON string values
2. Consider proper JSON formatting with indentation
3. Remove the 'properties=' prefix as it's not valid JSON syntax

Example:
```json
{
  "properties": {
    "owner": {
      "description": "Owner of the repository",
      "title": "Owner",
      "type": "string"
    },
    "repo": {
      "description": "Repository name",
      "title": "Repo",
      "type": "string"
    }
  },
  "title": "WebhookConfigSchema",
  "type": "object",
  "required": ["owner", "repo"]
}
```